spec/computable/registry/apply.spec.ts

```diff
@@ -142,16 +143,15 @@ describe('Registry: Apply', () => {
 
     // TODO the WS listener never fires if we use it here, investigate why
     it('should revert if applicationExpiry would overflow', async () => {
-      const BN = web3.utils.BN,
-        eth = web3.eth,
-        bigOne = new BN(1),
-        block:Block = await eth.getBlock(await eth.getBlockNumber())
+      const eth = web3.eth
+      const bigOne = new BigNumber(1)
+      const block:Block = await eth.getBlock(await eth.getBlockNumber())
 
       expect(block).toBeTruthy()
       // create an applyStageLen that, when added to current block time will be
       // greater than 2^256-1
-      const maxUint = new BN(2).pow(new BN(256)).sub(bigOne),
-        applyStageLen = maxUint.sub(new BN(block.timestamp)).add(bigOne),
+      const maxUint = new BigNumber(2).pow(256).minus(bigOne),
+        applyStageLen = maxUint.minus(new BigNumber(block.timestamp)).plus(bigOne),
```

Comment:
Yeah, it no longer plays nice. I think its types are messed up in the web3 export. It complains about not having a constructor anymore 🤷‍♂️ Rather than fight the type battle, I just updated it to use the actual bignumber lib.